e2e/rm_test.go

```diff
@@ -472,8 +472,8 @@ func TestVariadicMultipleLocalFilesWithDirectory(t *testing.T) {
 		0: equals(""),
 		1: equals("rm file1.txt"),
 		2: equals("rm file2.txt"),
-		3: equals("rm readme.md"),
-	}, sortInput(true))
+		3: equals("rm testdir/readme.md"),
+	}, strictLineCheck(true), sortInput(true))
```

Comment:
we can remove strictLineCheck(true). it's enabled by default.

Reply:
done